### DISCUSSION TITLE
Foundry DevOps and Anvil chain

### DISCUSSION THREAD

**Discussion by chuksgpfr** [Cyfrin/foundry-full-course-cu#2471] [Q&A] [answered]
Is there any reason foundry devops doesn't work with Anvil ?

`function run() external {
    console.log("Deploying NFT...");
    address recentlyDeployed = DevOpsTools.get_most_recent_deployment("NFT", block.chainid);
    console.log("recentlyDeployed", recentlyDeployed);
    mintNFTOnContract(recentlyDeployed);
  }`

I get the first log but not the second one.

PS: I logged because I kept getting the error `unknown selector 0xdb4235f6 for VmCalls`

**Comment by EngrPips** [comment]
Hello @chuksgpfr, Please show properly the script you are running and the output you get when you run the script.

**Comment by chuksgpfr** [comment, marked as answer]
Fixed this by upgrading foundry and anvil using `foundryup`

**Comment by EngrPips** [comment]
Awesome, Keep crushing it.
